lusidtools/cocoon/transaction_type_upload.py

```diff
@@ -58,3 +58,82 @@ def create_transaction_type_configuration(api_factory, alias, movements):
     )
 
     return response
+
+
+def replace_current_transaction_alias(
+    api_factory, updated_alias, movement, properties=None
+):
+
+    """
+    This function overrides the current transaction types alias with a new configuration
+
+    Parameters
+    ----------
+    api_factory : lusid.utilities.ClientApiFactory
+        The LUSID api factory to use
+    alias : lusid.models.TransactionConfigurationTypeAlias
+        An aliases with type and group
+    movements : list[lusid.models.TransactionConfigurationMovementDataRequest]
```

Comment:
Great having a docstring! Unfortunately these two parameter names don't match the names in the function. Please update.

Reply:
Ah, a copy and paste error from me. That's fixed!